2018/ffs-wp.md

```diff
@@ -0,0 +1,277 @@
+Mit anderen Worten: obige Quellen von Ressourcen sind nicht ausreichend.
+
+
+## Analyse existierender Geschäftsmodelle
+
+Wikipedia listet folgende [Geschäftsmodelle](https://en.wikipedia.org/wiki/Business_models_for_open-source_software)
+für "Open-Source-Software" auf:
+
+    3.1 Dual-licensing
+    3.2 Selling professional services
+    3.3 Selling of branded merchandise
+    3.4 Selling of certificates and trademark use
+    3.5 Selling software as a service
+    3.6 Partnership with funding organizations
+    3.7 Voluntary donations
+    3.8 Bounty driven development
+    3.9 Pre-order/crowdfunding/reverse-bounty model
+    3.10 Crowdsourcing
+    3.11 Advertising-supported software
+    3.12 Selling of optional proprietary extensions
+    3.13 Selling of required proprietary parts of a software product
+    3.14 Selling of proprietary update systems
+    3.15 Re-licensing under a proprietary license
+    3.16 Obfuscation of source code
+    3.17 Delayed open-sourcing
+    3.18 Open sourcing on end-of-life
+
+
+ Für die Projekte mit der Zielgruppe der Endbenutzer\*innen kommen bestimmte Modelle
+ nicht in Frage bzw. sind nicht wünschenswert (Unterschied zwischen Open-Source und freier Software).
+ Im Folgenden sind Ausschlussgründe und die davon betroffenen Geschäftsmodelle aufgelistet.
+
+
+- Nicht sinnvoll, Weil kein Weiterverkauf angestrebt wird
+    - Dual-licensing
+    - Re-licensing under a proprietary license
+- Zielkonflikt: Perfekte Software braucht keinen Support/Schulung
+    - Selling professional services
+- Problem der Datensouveränität.
+    - Selling software as a service
+
+
+- Widerspricht dem Ziel maximalen FLOSS-Anteils
+    - Selling of optional proprietary extensions
+    - Selling of required proprietary parts of a software product
+    - Selling of proprietary update systems
+    - Obfuscation of source code
+    - (Delayed open-sourcing)
+
+- Nicht nachhaltig
+    - Open sourcing on end-of-life
+
+Es bleiben also folgende Modelle übrig:
+
+    3.3 Selling of branded merchandise
+    3.4 Selling of certificates and trademark use
+    3.6 Partnership with funding organizations
+    3.7 Voluntary donations
+    3.8 Bounty driven development
+    3.9 Pre-order/crowdfunding/reverse-bounty model
+    3.10 Crowdsourcing
+    3.11 Advertising-supported software
+
+## Ideenskizze zur Kombination (einiger) dieser Methoden
+
+### Vorbemerkung
+"Voluntary donations" klingt, als wäre die Zahlung "gegen die ökonomische Vernunft" und allein
+aus irrationalem Altruismus oder Mitleid. Ohne tiefer in die Kritik des Modells des Homo-Oeconomicus
+einzusteigen, zwei Anmerkungen:
+
+- In vielen anderen Bereichen sind Menschen aber bereit, erhebliche Mittel "entgegen der ökonomischen Vernunft" auszugeben, Stichwort: Statuskonsum.
+- Wie oben schon erwähnt, kann die finanzielle Unterstützung eines Projektes von dem man individuell profitiert auch mit informiertem Eigeninteresse begründet werden.
```

Comment:
Die Finanzierung und das damit verbundene Feedback motiviert die Entwickler extrinsisch – das ist von enormer Bedeutung weil der intrinsischen Motivation meist durch die Erledigung der eigenen Anforderungen genüge getan ist.

Reply:
Das ist wohlgemerkt mitnichten entgegen ökonomischer Vernunft wenn ein Zweck verfolgt wird.
Ich würde in der Art argumentieren: die Community lebt von Beiträgen dritter; Menschen fühlen sich einer Sache verbunden und wollen ggf. daran im Rahmen ihrer Möglichkeiten "mitwirken", der nicht zu vernachlässigende Anteil derer ohne hinreichende (Programmier-)Kenntnisse kann sich aber anderweitig (Übersetzung, Dokumentation, Bugreports, …) und im Zweifel finanziell einbringen.
Eine möglichst niedrige Schwelle (z.B. Übersetzung via transifex, vielfältige oder einfach zu handhabende Zahlungsmöglichkeit) für dieses Einbringen vergrößert dabei die Wahrscheinlichkeit erheblich.